native/cocos/scene/Camera.cpp

```diff
@@ -218,6 +215,13 @@ void Camera::changeTargetWindow(RenderWindow *window) {
     }
 }
 
+void Camera::initGeometryRenderer() {
```

Comment:
Where is `initGeometryRenderer` invoked?

Reply:
It is invoked by users